docs/.snippets/code/build-on-polkadot/xcm/asset-transfer-api/reference/transfer-arg-opts.md

```diff
@@ -0,0 +1,104 @@
+Options for customizing the claim assets transaction. These options allow you to specify the transaction format, fee payment details, weight limits, XCM versions, and more.
+
+??? child "Show more"
+
+    `format` ++"T extends Format"++ 
+        
+    Specifies the format for returning a transaction.
+
+    ??? child "Type `Format`"
+
+        ```ts
+        export type Format = 'payload' | 'call' | 'submittable';
+        ```
+    
+    ---
+
+    `paysWithFeeOrigin` ++"string"++
+    
+    The Asset ID to pay fees on the current common good parachain. The defaults are as follows:
+
+        - Polkadot AssetHub - DOT
+        - Kusama AssetHub - KSM
+
+    ---
+
+    `paysWithFeeDest` ++"string"++
+    
+    Asset ID to pay fees on the destination parachain.
+
+    ---
+
+    `weightLimit` ++"{ refTime?: string, proofSize?: string }"++
+    
+    Custom weight limit option. If not provided, it will default to unlimited.
+
+    ---
+
+    `xcmVersion` ++"number"++
+    
+    Sets the XCM version for message construction. If this is not present a supported version will be queried, and if there is no supported version a safe version will be queried.
+
+    ---
+
+    `keepAlive` ++"boolean"++
+    
+    Enables `transferKeepAlive` for local asset transfers. For creating local asset transfers, if true this will allow for a `transferKeepAlive` as opposed to a `transfer`.
```

Comment:
```suggestion
    Enables `transferKeepAlive` for local asset transfers. For creating local asset transfers, if `true` this will allow for a `transferKeepAlive` as opposed to a `transfer`.
```